routersploit/interpreter.py

```diff
@@ -456,5 +457,11 @@ def command_help(self, *args, **kwargs):
     def command_exec(self, *args, **kwargs):
         os.system(args[0])
 
+    def command_search(self, *args, **kwargs):
+        for arg in args:
+            matches = [s for s in self.modules if arg in str(s)]
```

Comment:
`str(s)` is not necessary, `self.modules` is already list of strings, please take a look at method `__show_modules()`